terraform.tfvars

```diff
@@ -0,0 +1,7 @@
+env_code = "main"
+
+vpc_cidr = "10.0.0.0/16"
+
+public_cidr = ["10.0.0.0/24", "10.0.1.0/24"]
+
+private_cidr = ["10.0.2.0/24", "10.0.3.0/24"]
```

Comment:
Please add a newline to the end of this file

Reply:
newline as been appended to file terraform.tfvars